Directory.Build.targets

```diff
@@ -1,7 +1,15 @@
 <Project>
 
+  <PropertyGroup>
+    <IsSourceProject Condition="'$(IsSourceProject)' == '' and
+                                '$(IsTestProject)' != 'true' and
+                                '$(IsTestUtilityProject)' != 'true'">true</IsSourceProject>
+  </PropertyGroup>
+
+  <Import Project="$(RepositoryEngineeringDir)unsetispackable.targets" Condition="'$(UnsetIsPackableForNonSourceProjects)' == 'true' and '$(IsSourceProject)' != 'true'" />
   <Import Project="$(RepositoryEngineeringDir)resources.targets" />
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
   <Import Project="$(RepositoryEngineeringDir)testing\runsettings.targets" Condition="'$(IsTestProject)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)placeholderpackaging.targets" Condition="'$(IsPlaceholderTargetFramework)' == 'true'" />
   <Import Project="$(RepositoryEngineeringDir)packaging.targets" Condition="'$(IsPackable)' == 'true'" />
```

Comment:
Line 45-46 is importing eng/testing/skiptests.targets using the SkipTests condition.

Reply:
As mentioned above, `SkipTests`is still used.